server/src/helpers/character/DialogActionHelper.ts

```diff
@@ -19,6 +19,9 @@ export class DialogActionHelper extends BaseService {
 
   public async handleDialog(player: IPlayer, npc: INPC, command: string, callbacks): Promise<void> {
     const messages = await (npc as any).dialogParser.parse(command, { player, callbacks });
+    if ((messages || []).length === 0) {
+      this.game.messageHelper.sendLogMessageToPlayer(player, {message: this.getDefaultMessage(), from: npc.name}, [MessageType.NPCChatter]);
```

Comment:
to simplify this, instead of sending the message here, do `messages.push(this.getDefaultMessage()`. 

the above line will need to be changed to `    const messages = (await (npc as any).dialogParser.parse(command, { player, callbacks })) || [];`

Reply:
Thanks